examples/kafka/scramsecretassociation.yaml

```diff
@@ -0,0 +1,242 @@
+    region: us-east-2
+    secretArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: scram-secret-association
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: example1
+  namespace: upbound-system
+type: Opaque
+stringData:
+  json: '{"username":"uptest1","password":"password"}'
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: example2
+  namespace: upbound-system
+type: Opaque
+stringData:
+  json: '{"username":"uptest2","password":"password"}'
+
+---
+
+apiVersion: kms.aws.upbound.io/v1beta1
+kind: Key
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: scram-secret-association
+  name: example
+spec:
+  forProvider:
+    description: Example Key for MSK Cluster Scram Secret Association
+    region: us-east-2
+
+---
+
+apiVersion: secretsmanager.aws.upbound.io/v1beta1
+kind: Secret
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: scram-secret-association
+  name: example1
+spec:
+  forProvider:
+    kmsKeyIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: scram-secret-association
+    name: AmazonMSK_example1
+    region: us-east-2
+    recoveryWindowInDays: 0
+
+---
+
+apiVersion: secretsmanager.aws.upbound.io/v1beta1
+kind: Secret
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: scram-secret-association
+  name: example2
+spec:
+  forProvider:
+    kmsKeyIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: scram-secret-association
+    name: AmazonMSK_example2
+    region: us-east-2
+    recoveryWindowInDays: 0
+
+---
+
+apiVersion: secretsmanager.aws.upbound.io/v1beta1
+kind: SecretVersion
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: scram-secret-association
+  name: example1
+spec:
+  forProvider:
+    region: us-east-2
+    secretIdRef:
+      name: example1
+    secretStringSecretRef:
+      key: json
+      name: example1
+      namespace: upbound-system
+
+---
+
+apiVersion: secretsmanager.aws.upbound.io/v1beta1
+kind: SecretVersion
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: scram-secret-association
+  name: example2
+spec:
+  forProvider:
+    region: us-east-2
+    secretIdRef:
+      name: example2
+    secretStringSecretRef:
+      key: json
+      name: example2
+      namespace: upbound-system
+
+---
+
+apiVersion: kafka.aws.upbound.io/v1beta1
+kind: Cluster
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+    upjet.upbound.io/manual-intervention: "This resource needs valid Configuration arn and revision."
```

Comment:
This is the same workaround @turkenf used in #864. I've got an idea that might work, which I'll post in a separate PR.